README*.md

```diff
@@ -1,3 +1,8 @@
+\* = `plotlyupdate.m` is currently disabled. Please download the latest version 
```

Comment:
It will show up as `README*.md` -- we could def rename it to something else though!